clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -217,7 +221,11 @@ public Struct parseResponse(short version, ByteBuffer buffer) {
     DESCRIBE_CLIENT_QUOTAS(48, "DescribeClientQuotas", DescribeClientQuotasRequestData.SCHEMAS,
             DescribeClientQuotasResponseData.SCHEMAS),
     ALTER_CLIENT_QUOTAS(49, "AlterClientQuotas", AlterClientQuotasRequestData.SCHEMAS,
-            AlterClientQuotasResponseData.SCHEMAS);
+            AlterClientQuotasResponseData.SCHEMAS),
+    BROKER_HEARTBEAT(50, "BrokerHeartbeat", BrokerHeartbeatRequestData.SCHEMAS,
+            BrokerHeartbeatResponseData.SCHEMAS),
+    CONTROLLER_HEARTBEAT(51, "ControllerHeartbeat", ControllerHeartbeatRequestData.SCHEMAS,
+            ControllerHeartbeatResponseData.SCHEMAS);
```

Comment:
50 and 51 are declared in KIP-554 for {List,Alter}ScramUsers, respectively.

Reply:
Yeah.  We'll increment this as needed when that KIP gets accepted and we merge into the branch.